systemtests/utils/utils.go

```diff
@@ -154,6 +153,8 @@ func StartNetPlugin(t *testing.T, nodes []TestbedNode, nativeInteg bool) {
 
 // StartNetmasterWithFlags starts netplugin on specified testbed nodes with specified flags
 func StartNetmasterWithFlags(t *testing.T, node TestbedNode, flags map[string]string) {
+	time.Sleep(5 * time.Second)
```

Comment:
Ah, I forgot to add this additional context:

so at some point during the b/t cycle for these features (packer, and this code) there was a largeish issue with vbox 5.0.0's networking which got us stuck plumbing the wrong interfaces. I think you hit this last week sometime, so maybe it's an artifact of that.